lib/redisWrapper.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+var redisWrapper = module.exports = {};
+var Q = require('q');
+var redis = require('redis');
+var _ = require('lodash');
+var redisClient;
+var separator = ':';
+var wildcard = '*';
+
+redisWrapper.initialise = function(serverConfig) {
+  var deferred = Q.defer();
+  if (_.isEmpty(serverConfig)) {
+    deferred.reject('no serverConfig passed');
+  } else {
+    redisClient = redis.createClient(serverConfig);
+    redisClient.on('ready', function () {
+      deferred.resolve();
+    });
+    // if there was error connecting to redis server
+    redisClient.on('error', function () {
```

Comment:
We don't have test coverage on this function, meaning we get 'ERROR: Coverage for functions (96%) does not meet global threshold (100%)' when running the tests. Perhaps we should update the .istanbul.yml